plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -391,6 +392,24 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderInstances(prop, service) {
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+    const overview = tasksRunning === instancesCount
+      ? ` ${tasksRunning}`
+      : ` ${tasksRunning}/${instancesCount}`;
+
+    const content = instancesCount == null || service.list
```

Comment:
What is the meaning of `service.list`? It's not part of the struct.

Reply:
whenever there is a path/folder `list` will exist containing the child paths.

I guess I can add to the struct since doesn't exist. Thanks.